src/pybind/mgr/cephadm/ssh.py

```diff
@@ -139,31 +139,47 @@ def remote_connection(self,
 
     async def _execute_command(self,
                                host: str,
-                               cmd: List[str],
+                               cmds: List[str],
                                stdin: Optional[str] = None,
                                addr: Optional[str] = None,
                                log_command: Optional[bool] = True,
                                ) -> Tuple[str, str, int]:
+
+        def host_to_addr(host: str) -> str:
```

Comment:
Why define a closure function if it exists only to be a try-except block? If it's called only once, I think it would be clearer to simply write the `address` assignment like it was before but moved out of the except blocks.